docs/source/models_and_datasets.md

```diff
@@ -17,4 +17,15 @@ limitations under the License.
 
 # Morpheus Models
 
-Morpheus comes with a number of pretrained models with corresponding training, validation scripts, and datasets. The latest release of these models can be found [here](https://github.com/nv-morpheus/Morpheus/blob/-/models).
\ No newline at end of file
+Morpheus comes with a number of pretrained models with corresponding training, validation scripts, and datasets. The latest release of these models can be found [here](https://github.com/nv-morpheus/Morpheus/blob/-/models).
+
+|Model|GPU Mem Req|Description|
+|-----|-----------|-----------|
+|[Anomalous Behavior Profiling (ABP)](https://github.com/nv-morpheus/Morpheus/blob/-/models#anomalous-behavior-profiling-abp)|2015MiB|This model is an example of a binary classifier to differentiate between anomalous GPU behavior such as crypto mining / GPU malware, and non-anomalous GPU-based workflows (e.g., ML/DL training). The model is an XGBoost model.|
```

Comment:
I think we should create a `docs/source/models` directory add a symlink in there to `../../../models/README.md` allowing us to link to that doc as a relative link.

This way we are always directing the reader to the version of that doc for their version.

Reply:
I thought symlink didn't play nice with Sphinx when it came to relative links? I looked into doing this when I first added the modeling documentation to `docs/source`. That's also why I did the hard-coded links to the model directories.

Reply:
If you take a look in the `docs/source/examples/abp_nvsmi_detection` dir you will see some symlinks to files in the `examples/abp_nvsmi_detection` dir:
```
$ ls -l
total 0
lrwxrwxrwx 1 dagardner dagardner 53 Mar  6 14:22 pipeline.png -> ../../../../examples/abp_nvsmi_detection/pipeline.png
lrwxrwxrwx 1 dagardner dagardner 50 Mar  6 14:22 README.md -> ../../../../examples/abp_nvsmi_detection/README.md
```

The symlinks *should* work, so long as they are relative, and are checked into git.